README.md

```diff
@@ -96,7 +96,7 @@ The server pod is already running the accelerated `sfnt-pingpong` instance.
 
 Run the client:
 ```
-sh-4.4# LD_PRELOAD=/opt/onload/usr/lib64/libonload.so ./sfnt-pingpong udp 198.19.0.1
+sh-4.4# ./sfnt-pingpong udp 198.19.0.1
```

Comment:
Can we make sure this main top-level example uses the latency profile to get good out-of-the-box performance?